integration_tests/.#.#.#camera_settings.cpp

```diff
@@ -0,0 +1 @@
+omar@linux-4-102.722:1545258157
```

Comment:
Please remove this file.

And to ignore them in the future you can setup a global gitignore file for your machine:
https://stackoverflow.com/questions/7335420/global-git-ignore